docs/controllers/services/annotations.md

```diff
@@ -64,6 +64,10 @@ Specify whether the DigitalOcean Load Balancer should pass encrypted data to bac
 
 Specifies the certificate ID used for https. To list available certificates and their IDs, install [doctl](https://github.com/digitalocean/doctl) and run `doctl compute certificate list`.
 
+## service.beta.kubernetes.io/do-loadbalancer-hostname
+
+Specifies the hostname used for the LB `status.Hostname` instead of assigning `status.IP` directly. This can be used to avoid this [iptables rule being written by kube-proxy](https://github.com/kubernetes/kubernetes/blob/release-1.15/pkg/proxy/iptables/proxier.go#L958-L979) for the IP. This is useful when using SSL or proxy protocol on the LB, which will break if the request doesn't goes through it, but instead is routed directly in-cluster.
```

Comment:
IMO, pointing at the iptables rules is a bit too technical / detailed. I'd keep it at the behavioral level and reference the upstream issue. Suggestion:

```suggestion
Specifies the hostname used for the Service `status.Hostname` instead of assigning `status.IP` directly. This can be used to avoid bypassing the external LB when requests originate at a pod in order to enforce SSL or proxy protocol processing.

See also [Kubernetes issue 66607](github.com/kubernetes/kubernetes/issues/66607).
```